packages/ubuntu_desktop_installer/lib/pages/who_are_you/who_are_you_page.dart

```diff
@@ -0,0 +1,368 @@
+                            value: LoginStrategy.autoLogin,
+                            groupValue: _whoAreYouModel.loginStrategy,
+                            onChanged: (_) => setState(() {
+                                  _whoAreYouModel.loginStrategy =
+                                      _whoAreYouModel.loginStrategy;
+                                })),
+                        Padding(
+                          padding: const EdgeInsets.all(8.0),
+                          child: Text(lang.whoAreYouPageLoginStrategyAutoLogin),
+                        )
+                      ],
+                    ),
+                    Row(
+                      children: [
+                        Radio<LoginStrategy>(
+                            value: LoginStrategy.requirePassword,
+                            groupValue: _whoAreYouModel.loginStrategy,
+                            onChanged: (_) => setState(() {
+                                  _whoAreYouModel.loginStrategy =
+                                      LoginStrategy.requirePassword;
+                                })),
+                        Padding(
+                          padding: const EdgeInsets.all(8.0),
+                          child: Text(
+                              lang.whoAreYouPageLoginStrategyRequirePassword),
+                        )
+                      ],
+                    ),
+                  ],
+                ),
+              ),
+              actions: <WizardAction>[
+                WizardAction(
+                  label: lang.backButtonText,
+                  onActivated: Navigator.of(context).pop,
+                ),
+                WizardAction(
+                  label: lang.startInstallingButtonText,
+                  highlighted: true,
+                  onActivated: () async {
+                    if (_realNameFormKey.currentState!.validate() &&
+                        _computerNameFormKey.currentState!.validate() &&
+                        _usernameFormKey.currentState!.validate() &&
+                        _passwordFormKey.currentState!.validate() &&
+                        _confirmPasswordFormKey.currentState!.validate()) {
+                      final client =
+                          Provider.of<SubiquityClient>(context, listen: false);
+
+                      final key = encrypt.Key.fromLength(32);
+                      final iv = encrypt.IV.fromLength(16);
+                      final encrypter = encrypt.Encrypter(encrypt.AES(key));
+
+                      final encrypted =
+                          encrypter.encrypt(_whoAreYouModel.password, iv: iv);
+
+                      await client.setIdentity(IdentityData(
+                          hostname: _computerNameController.text,
+                          realname: _realNameController.text,
+                          username: _usernameController.text,
+                          cryptedPassword: encrypted.toString()));
```

Comment:
can we have this logic in the model instead? `WhoAreYouModel.saveIdentity()` - a counterpart for `loadIdentity` that is called at init